cpp/src/binaryop/binaryop.cpp

```diff
@@ -52,10 +52,10 @@ namespace binops {
 /**
  * @brief Computes output valid mask for op between a column and a scalar
  */
-rmm::device_buffer scalar_col_valid_mask_and(column_view const& col,
-                                             scalar const& s,
-                                             rmm::cuda_stream_view stream,
-                                             rmm::mr::device_memory_resource* mr)
+rmm::device_buffer scalar_col_bitmask_and(column_view const& col,
```

Comment:
We have a subtle distinction between valid masks and bitmasks. This case is a valid mask. "Bitmasks" are never `nullptr` but "valid masks" might not be allocated (just `nullptr`). This checks if the column is nullable so it's dealing with valid masks.